site/content/docs/5.3/content/tables.md

```diff
@@ -14,6 +14,12 @@ Due to the widespread use of `<table>` elements across third-party widgets like
 
 Using the most basic table markup, here's how `.table`-based tables look in Boosted.
 
+<!-- Boosted mod : design callout -->
```

Comment:
You can remove this comment which is not used for other callouts. The shortcode is rather straightforward.